source/_integrations/chacon_dio.markdown

```diff
@@ -20,6 +22,7 @@ This {% term integrations %} gives you access to the Wi-Fi connection so that Ho
 There is currently support for the following information within Home Assistant:
 
 - Cover devices (get statuses, move up, down, stop, and move to a given percentage)
+- Switch devices that can be lights or plugs (get statuses, turn on and off)
```

Comment:
should have mentioned this already in the previous PR. But as this integration is getting more elements, it should have separate sections for each platform. Please add a separate section called Switches after the config flow, listing the switch names together with a description. Please do the same for the cover. This could be a separate PR.

Reply:
Done just now. Is it OK for you ?
For a separate PR, just for cover waiting the switch is merged, I suppose it is to be done on the current branch, is it correct ?